pytest_terraform/tf.py

```diff
@@ -116,6 +116,34 @@ def _run_cmd(self, args):
         run_cmd(args, cwd=cwd, stderr=subprocess.STDOUT, env=env)
 
 
+class TerraformStateJson(UserString):
+    @classmethod
+    def from_obj(cls, state):
+        s = cls("")
+        s.dict = state
+        return s
+
+    def update(self, state):
+        if not isinstance(state, str):
+            raise ValueError(f"{state} is not a string")
+
+        self.data = str(state)
+
+    def load(self, state):
```

Comment:
the type info associated to these is a bit unclear from the names .. ie load a string takes a dictionary, update a string takes a string, its not really obvious to a caller which is which and the types they take. lets be explicit via name and change  load to say load_data ?

Reply:
Yeah, totally. I wrestled with these a bit I'll make a more concise name

Reply:
I went with `update` as string and `update_dict` (take a dict and update string state with it)